EMBED.md

```diff
@@ -14,18 +14,18 @@ This will show the player inside the iframe and will automatically resize.
 
 Both targetId and channelId will be provided to you on the creation of your Zender account. They can also be found through the Zender admin console.
 
-# Zender Lobby mechanism
-By default when no streams are public there will nothing to see.
+### Zender Lobby mechanism
+By default no streams are public and there will be nothing to see.
```

Comment:
This changes the meaning a bit: it's an admins choice whether any stream is public.
I'd go with:
When there is no lobby stream and no streams are public, there will be nothing to see (or show?).

Reply:
Just also removed the by default since that suggested us describing the initial configuration